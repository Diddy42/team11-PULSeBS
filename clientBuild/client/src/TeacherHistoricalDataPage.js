import React from 'react';

class TeacherHistoricalDataPage extends React.Component{
    render(){
        return <>
            <h1>Historical data</h1>
        </>
    }
}

export default TeacherHistoricalDataPage;