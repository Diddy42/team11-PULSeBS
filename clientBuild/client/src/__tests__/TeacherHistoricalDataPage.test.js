import TestRenderer from 'react-test-renderer';
import React from 'react';
import TeacherHistoricalDataPage from '../TeacherHistoricalDataPage';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('renders learn react link', () => {
    const tree = TestRenderer.create(<TeacherHistoricalDataPage/>).toJSON();
    expect(tree).toMatchSnapshot();
  });