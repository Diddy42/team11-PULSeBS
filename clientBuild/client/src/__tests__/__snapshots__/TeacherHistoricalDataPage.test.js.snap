// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-2 mt-3 col"
    >
      <div
        className="list-group"
        onKeyDown={[Function]}
      >
        <button
          className="list-group-item active list-group-item-action"
          data-rb-event-key={null}
          disabled={false}
          onClick={[Function]}
        >
          Lecture
        </button>
        <button
          className="list-group-item list-group-item-action"
          data-rb-event-key={null}
          disabled={false}
          onClick={[Function]}
        >
          Week
        </button>
        <button
          className="list-group-item list-group-item-action"
          data-rb-event-key={null}
          disabled={false}
          onClick={[Function]}
        >
          Month
        </button>
      </div>
    </div>
    <div
      className="col-8 mt-3 col"
    />
    <div
      className="col-2 mt-3 col"
    >
      <div
        className="list-group"
        onKeyDown={[Function]}
      />
    </div>
  </div>
</div>
`;
